# WIP - Help needed

URL customizations requires to comprehend the scope to handle the documentation for this
and other projects heavily customizing and rebranding Firefox like Abrowser does.

This patch documents how to handle custom URLs to point to a desired page (initially).

It replaces,

* is="moz-support-link"
* support-page="..."

to customize the default URL, making sure there is an id for l10n field,

* data-l10n-id="..."

so the corresponding message is displayed as it seems to be linked on some cases
with is="" and support-page="..."

Cheers!

diff --git a/browser/components/preferences/privacy.inc.xhtml b/browser/components/preferences/privacy.inc.xhtml_
index 77ea8f5d..62c3ce8e 100644
--- a/browser/components/preferences/privacy.inc.xhtml
+++ b/browser/components/preferences/privacy.inc.xhtml
@@ -372,10 +372,7 @@
           support-page="global-privacy-control" />
     </hbox>
     <hbox id="doNotTrackBox" flex="1" align="center" hidden="true">
-      <html:a is="moz-support-link"
-          id="doNotTrackRemoval"
-          support-page="how-do-i-turn-do-not-track-feature"
-          data-l10n-id="do-not-track-removal" />
+      <html:a class="learnMore" href="https://trisquel.info/en/wiki/abrowser-help" target="_blank"/>
     </hbox>
   </vbox>
 </groupbox>
@@ -388,11 +385,10 @@
     <vbox flex="1">
       <description class="description-with-side-element description-deemphasized" flex="1">
         <html:span id="totalSiteDataSize"></html:span>
-        <html:a is="moz-support-link"
-                id="siteDataLearnMoreLink"
-                data-l10n-id="sitedata-learn-more"
-                support-page="storage-permissions"
-        />
+        <html:a id="doNotTrackLearnMoreLink"
+            href="https://trisquel.info/en/wiki/abrowser-help"
+            data-l10n-id="do-not-track-learn-more"
+            target="_blank"/>
       </description>
       <hbox flex="1" id="deleteOnCloseNote" class="info-box-container smaller-font-size">
         <hbox class="info-icon-container">
